modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -1710,9 +1710,11 @@
 	end
 
 	function m.importNuGetTargets(prj)
-		for i = 1, #prj.nuget do
-			local targetsFile = nuGetTargetsFile(prj, prj.nuget[i])
-			p.x('<Import Project="%s" Condition="Exists(\'%s\')" />', targetsFile, targetsFile)
+		if _ACTION < "vs2017" then
```

Comment:
This `if _ACTION < "vs2017" then` check will also have to be removed for the same reason.